todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskActivity.java

```diff
@@ -55,19 +55,27 @@ protected void onCreate(Bundle savedInstanceState) {
                 (AddEditTaskFragment) getSupportFragmentManager().findFragmentById(R.id.contentFrame);
 
         String taskId = getIntent().getStringExtra(AddEditTaskFragment.ARGUMENT_EDIT_TASK_ID);
-
+        
+        boolean hasExtra = getIntent().hasExtra(AddEditTaskFragment.ARGUMENT_EDIT_TASK_ID);
```

Comment:
Sure, thanks for pointing it out, I am new to open source contribution, I have made the changes.
:)